compiler/plugins/input/StableHLO/Conversion/Preprocessing/test/stablehlo_to_stablehlo.mlir

```diff
@@ -341,28 +341,6 @@ func.func @dot_basic(%arg0: tensor<4x4xf16>, %arg1: tensor<4x4xf16>) -> tensor<4
 
 // -----
 
-// CHECK-LABEL: @convolution
-// CHECK-SAME:     (%[[ARG0:.+]]: tensor<{{.+}}xbf16>, %[[ARG1:.+]]: tensor<{{.+}}xbf16>)
-func.func @convolution(%arg0: tensor<16x32x256xbf16>, %arg1: tensor<1x256x256xbf16>) -> tensor<16x32x256xf32> {
-  %cast = stablehlo.convert %arg0 : (tensor<16x32x256xbf16>) -> tensor<16x32x256xf32>
-  // CHECK: %[[CONV:.+]] = stablehlo.convolution(%[[ARG0]], %[[ARG1]])
-  // CHECK-SAME: -> tensor<16x32x256xf32>
-  %0 = "stablehlo.convolution"(%cast, %arg1) {
```

Comment:
Please at least put context from commit messages in the PR description:
> Removed a convolution tests with different types. This test is
deprecated since it is illegal as per rule C27 in the stablehlo
spec: https://github.com/openxla/stablehlo/blob/main/docs/spec.md#convolution

For this though, we should have _some_ test for convolution. If this particular test was not formed well then we should replace it with one that is.

Reply:
Fixed.